src/services/models/packages.ts

```diff
@@ -149,6 +157,7 @@ export default function packagesModel(sequelize: Sequelize) {
         allowNull: true,
         type: DataTypes.STRING,
       },
+      license_detections: jsonDataType("license_detections"),
```

Comment:
Was this the source of the occasional sqlite error for packages?

Reply:
yes